cfpb/cfgov-refresh
# Removing Bower

Removing Bower in a move towards simplifying the front-end build process.

## Changes

- Modified `README.md`, `.travis.yml`,  `ansible/vars/app.yml`, `docs/front-end-build.md` to remove references  to Bower.

- Modified `frontend.sh` to remove Bower from the `frontend.sh` process.

- Modifed `gulp/config.js` to do the following:
 
  - Remove  unused assets:  the `/box-sizing-polyfill/boxsizing.htc` file didn't exist although it's being referenced in CSS file. Removed `/slick-carousel/slick/slick.css.map` as it's no longer being generated with `slick-carousel` version 1.6.  
  - Switch references from the `vendor` directory to the `node_modules` directory.


## Removals

- Removed `Bower.json` as it's no longer necessary.


## Testing

- Run `./frontend.sh`
- Verify that the necessary files are in your `static_built` directory.
- Browse the site; observe that there are no errors in the console and it operates as expected.

## Review

- @anselmbradford 
- @Scotchester 
- @jimmynotjim 
- @contolini 

## Notes

- `gulp copy` doesn't produce errors when the src files don't exist. 

## Checklist
